components/inlinealert/index.css

```diff
@@ -10,81 +10,121 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
-@import "../vars/css/components/spectrum-inlinealert.css";
-
 .spectrum-InLineAlert {
-  --spectrum-inlinealert-neutral-title-height: 38px;
-  --spectrum-inlinealert-neutral-corner-radius: 4px;
-
-  /* DNA */
-  --spectrum-inlinealert-neutral-border-width: 2px;
+  /* Static properties */
+  --spectrum-inlinealert-font-size: 14px;
+  --spectrum-inlinealert-font-weight: 700;
+  --spectrum-inlinealert-font-style: normal;
+  --spectrum-inlinealert-line-height: 14px;
+
+  /* Size*/
```

Comment:
✨ Custom properties are so nicely organized!
Nit: Line 20 and 27 lack a space before their closing */